libs/api-interfaces/src/lib/aggregate-views/view-models/note/note.view-model.interface.ts

```diff
@@ -1,7 +1,8 @@
 import { IBaseViewModel } from '../base.view-model.interface';
+import { ResourceType } from '../resources';
 import { IEdgeConnectionMember } from './edge-connection-member.interface';
 
 export interface INoteViewModel extends IBaseViewModel {
     note: string;
-    relatedResources: IEdgeConnectionMember[];
+    relatedResources: IEdgeConnectionMember<string, ResourceType>[];
```

Comment:
We should strongly consider getting rid of the generic in the IEdgeConnectionMember now that we have moved the enum to the api-interfaces anyway. That was the only reason for having the generic- being able to inject the union type from the num from somewhere with the extra knowledge.

Reply:
done. note that we will need to expose the context types from api-interfaces at some point